R/pkg/inst/profile/shell.R

```diff
@@ -17,6 +17,7 @@
 
 .First <- function() {
   home <- Sys.getenv("SPARK_HOME")
+  sparkVer <- Sys.getenv("SPARK_VERSION")
```

Comment:
I'm not sure when this environment variable is set. We can instead call the java method `sc.version` after creating the spark context. To do that you can do something like 
```
sparkVer <- SparkR:::callJMethod(sc, "version")
```